config.template.py

```diff
@@ -18,7 +19,7 @@ class Config(object):
     # Unique prefix that defines VM name space for this Tango
     # version. When working in development, this prefix should be your
     # unique identifier. The "prod" prefix is reserved for production
-    PREFIX = "local"
+    PREFIX = "prod" if os.getenv("DOCKER_DEPLOYMENT").lower() == "true" else "local"
```

Comment:
This would break if the env var was not defined, as it becomes None.

I also think it would be more extensible if we allowed the DOCKER_DEPLOYMENT variable to directly specify the prefix, so that if the user wants to change it to development or test they can just do that easily by changing the docker-compose file